util/check-format-test-negatives.c

```diff
@@ -150,6 +150,10 @@ int f(void) /*
         hanging_stmt;
 }
 
+/* not: constant on LHS of comparison or assignment operator */
```

Comment:
drive-by remark: this formatting is more often used for "note:", so if the intent is to say that this is not supposed to trigger the "constant on LHS" check, maybe write it as something like "regression test: do not trigger constant on LHS of comparison or assignment operator here"

Reply:
Yeah, the "not: " was a bit hard to understand here. 
Changed to "should not trigger", but did not include "regression test" here because
essentially all of `util/check-format-test-negatives.c` is some kind of regression test.